### PR TITLE
feat: setup changeset

### PR DESCRIPTION
**Pull request by tdeekens**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,58 +1,72 @@
...
         run: yarn install --frozen-lockfile
-      - name: Publish to next dist-tag
-        if: success()
+
+      - name: Creating .npmrc
         run: |
-          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
-          yarn release:next
+          cat << EOF > "$HOME/.npmrc"
+            email=npmjs@commercetools.com
```
Comment (emmenko) [inline .github/workflows/release.yml:54]: 🙂

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,58 +1,72 @@
...
+            //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          EOF
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        # uses: changesets/action@master
+        uses: emmenko/action@nm-fix-npmrc-overwrite
```
Comment (emmenko) [inline .github/workflows/release.yml:63]: PS: my PR got merged, you can use `changesets/action@master`

Comment (tdeekens) [reply .github/workflows/release.yml:63]: Yeah. Thanks. We need to use that on app-kit too. The branch was deleted. Breaks stuff :).  Pushed to master.

File: .github/workflows/release.yml (hunk 1 above)
Comment (tdeekens) [reply .github/workflows/release.yml:54]: Danke.

File: .github/workflows/release.yml (hunk 2 above)
Comment (emmenko) [reply .github/workflows/release.yml:63]: Ah right :)

**Pull request merged**